lib/bash/report_mysql_info.sh

```diff
@@ -1014,6 +1014,18 @@ section_innodb () {
    local bp_free="$(get_var Innodb_buffer_pool_pages_free "$status_file")"
    local bp_dirt="$(get_var Innodb_buffer_pool_pages_dirty "$status_file")"
    local bp_fill=$((${bp_pags} - ${bp_free}))
+
+   # dynamically allocate variable name - transaction_isolation
+   local mysql_version=$(get_var version "$variables_file" | awk -F'-' '{print $1}')
+   local transaction_isolation_var="tx_isolation"
+   version_greater_equal() { # Function to compare versions
```

Comment:
Why don't you compare version strings like we do on line 229 of `lib/bash/collect_mysql_info.sh`:
`if [ "$version" '<' "8.0" ]; then`

Reply:
i thought that it could be good if we specify exact version like 5.7.20,  due to it is not a number we cant able to use `-ge`, is it ok if i use it like `if ["$mysql_version" > "5.7.19"]`